src/Pages/About/About.js

```diff
@@ -1,10 +1,73 @@
 import React from 'react';
+import Navigationbar from '../../Components/Navigationbar';
+import Sterling from '../../Images/SterlingBit.svg';
+import Blake from '../../Images/BlakeBit.svg';
+
+import './About.css';
 
 const About = () => {
 
     return (
-        <div>
-            <p>About page</p>
+
+        <div className='about-page-container'>
+            <div className='header-page'>
+                <Navigationbar />
+                <div className='header-container'>
+                    <h1 className = 'what-to-say'>
+                        <p>Knowing What</p>
+                        <p>to Say is Hard</p>
+                    </h1>
+                    <h4 className = 'header-quote'>
+                        <p>S&B Cards provides you with the</p>
+                        <p>inspiration for unique card signatures</p>
+                        <p>for all card writing occasions.</p>
+                    </h4>
+                    <button className="meet-the-team-button">Meet the Team</button>
+                </div>
+                <p className='empty'></p>
```

Comment:
Would it work the same if this empty <p> tag is removed and the div is given margin-bottom: 20%; instead?

Reply:
I tried doing this but it doesn't seem to work. It causes the background image to only cover to the end of the button rather than extending past the button. Ill continue looking at other possible solutions.